internal/bminventory/inventory.go

```diff
@@ -2987,6 +2987,31 @@ func (b *bareMetalInventory) processDiskSpeedCheckResponse(ctx context.Context,
 	return b.hostApi.SetDiskSpeed(ctx, h, diskPerfCheckResponse.Path, diskPerfCheckResponse.IoSyncDuration, exitCode, nil)
 }
 
+func (b *bareMetalInventory) updateDomainNameResolutionResponse(ctx context.Context, host *models.Host, domainResolutionResponseReporter string) error {
```

Comment:
What's the reason for the word `Reporter` in the last parameter? I think it should just be `domainResolutionResponse`

Reply:
changed to domainResolutionResponseJson